tests/ui_tools/test_buttons.py

```diff
@@ -92,6 +94,32 @@ def test_text_content(self, mocker,
         assert len(text[0]) == len(expected_text) == (width - 1)
         assert text[0] == expected_text
 
+    def test_update_count(self, mocker):
+        """Make sure update_count is called with correct parameters"""
+        show_function = mocker.Mock()
+        top_button = TopButton(controller=mocker.Mock(),
```

Comment:
The goal of the test I added on lines 112-121 was to make sure that when the update_count function was called on the starred_button that it used the correct call args...I guess it would be more logical to test the StarredButton.__init__ function itself since that's where the actual change to the count_theme is taking place?